src/jvmMain/kotlin/org/cqfn/diktat/demo/controller/DemoController.kt

```diff
@@ -48,19 +49,21 @@ class DemoController {
         val codeFix = CodeFix(codeForm.initialCode!!, codeFormHtml.ruleSet[0])
         val file = getDemoFile()
         file.writeText(codeForm.initialCode!!)
-        try {
+        val result = runCatching {
             if (codeForm.fix) {
                 codeForm.fixedCode = codeFix.fix(file.absolutePath)
             } else if (codeForm.check) {
                 codeForm.fixedCode = codeForm.initialCode
                 codeFix.check(file.absolutePath)
             }
-        } catch (e: ParseException) {
-            codeForm.warnings = listOf(e.toString())
         }
-        codeForm.warnings = codeFix.listOfWarnings
-                .map { "Warn (${it.line}:${it.col}) ${it.detail}" }
-                .map { it.replace(file.absolutePath, "\"example_file_name\"") }
+        when {
+            result.isSuccess -> codeForm.warnings = codeFix.listOfWarnings
```

Comment:
move formatting to some method pls if possible

Reply:
Done